src/modules/GroupChannel/components/MessageTemplateWrapper/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { BaseMessage } from '@sendbird/chat/message';
+import MessageTemplate, { MessageTemplateProps } from '../../../../ui/MessageTemplate';
+import { MessageProvider } from '@sendbird/react-uikit-message-template-view';
```

Comment:
Would it be possible to rename this to 'MessageTemplateProvider' to avoid confusion with the 'MessageProvider' in the UIKit?
```suggestion
import { MessageProvider as MessageProvider } from '@sendbird/react-uikit-message-template-view';
```

Reply:
good point, done!